tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": false,
```

Comment:
just remove line to get back to default - there is an issue #99 for compiler options